### PR TITLE
feat(sdk-core): add more whitelisted build params

### PR DESCRIPTION
**Pull request by davidkaplanbitgo**
The extra build params added:

```
[
  'addressType',          'apiVersion',
  'consolidateAddresses', 'cpfpFeeRate',
  'cpfpTxIds',            'gasLimit',
  'hopParams',            'idfSignedTimestamp',
  'idfUserId',            'idfVersion',
  'maxFee',               'nonce',
  'preview',              'receiveAddress',
  'reservation',          'strategy',
  'nonParticipation',     'validFromBlock',
  'validToBlock',         'messageKey',
  'eip1559',              'keyregTxBase64',
  'closeRemainderTo',     'tokenName',
  'enableTokens',         'emergency'
]
```

The problem is the cpfp parameters not being passed through, but there is also not a real reason to not use the full list

BG-76974

### CODE REVIEW

**Review by brandonblack — DISMISSED** [review]

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 1]
```diff
@@ -2053,37 +2053,7 @@ export class Wallet implements IWallet {
       return this.sendManyTss(params);
     }
 
-    const selectParams = _.pick(params, [
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:2056]: `prebuildWhitelistedParams` is missing `comment`, `otp`, and `hop`.

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

**Review by CapnMigraine — DISMISSED** [review]

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 2]
```diff
@@ -168,6 +168,9 @@ export class Wallet implements IWallet {
       'consolidateAddresses',
       'cpfpFeeRate',
       'cpfpTxIds',
+      'comment',
```
Comment (brandonblack) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:171]: Do these belong on the build params?

**Review by brandonblack — APPROVED** [review]

File: modules/sdk-core/src/bitgo/wallet/wallet.ts (hunk 2 above)
Comment (davidkaplanbitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts]: moved to within

Comment (mmcshinsky-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts]: They were part of `selectParams`, so whatever the solution, they need to be included.

Comment (davidkaplanbitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts]: they are

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**